extensions/amp-access/0.1/amp-access.js

```diff
@@ -149,6 +166,9 @@ export class AccessService {
 
     // Start authorization XHR immediately.
     this.runAuthorization_();
+
+    // Wait for the "view" signal.
+    this.scheduleView_();
```

Comment:
Good question. Do you think we should? Seems like we shouldn't on the first read. E.g. if the person opened a document, then switched to a new tab and then came back to the previous tab.